package.json

```diff
@@ -53,12 +54,20 @@
     "start-hoc": "cd packages/light-hoc && yarn start",
     "start-react": "cd packages/fether-react && yarn start",
     "start-ui": "cd packages/fether-ui && yarn start",
-    "test": "lerna run test --parallel"
+    "test": "CI=true lerna run test --parallel"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "FILES=`git diff --staged --name-only HEAD | grep -E '.js$|.ts$'`; [ -z \"$FILES\" ] && exit 0; yarn lint-files $FILES; git add $FILES",
+      "pre-rewrite": "FILES=`git diff --staged --name-only HEAD | grep -E '.js$|.ts$'`; [ -z \"$FILES\" ] && exit 0; yarn lint-files $FILES; git add $FILES"
```

Comment:
Is pre-rewrite a valid hook? https://git-scm.com/docs/githooks Or a husky addition?